radio/src/targets/common/arm/stm32/bootloader/boot.cpp

```diff
@@ -213,6 +213,17 @@ int main()
   pwrInit();
   pwrOff();
 
+  // wait a bit for the inputs to stabilize...
```

Comment:
It has been removed on purpose and now it will break other boards. please add if defined(HORUS) here

Reply:
It was for the X9 lite, not for Horus. I could not start the bootloader after setting the power-up delay to 0.

Reply:
anyways, it seems it works for me now without that busy loop (X9 lite).

Reply:
I think the code should be there for HORUS